File: Level 2/2016년/2016년.cpp
#include<iostream>
#include<string>
#include<vector>

using namespace std;
string getDayName(int a,int b)
{
    vector <string> day = {"FRI", "SAT", "SUN","MON", "TUE", "WED", "THU"};
    int i = 0;
    int total_days = - 1;
    int date = 0;
    for (i = 1; i < a; i++){
        if (i != a){
            if ((1 <= i && i <= 7 && i % 2 == 1) || (8 <= i && i <= 11 && i % 2 == 0)) {
                total_days += 31;
            }
            else if ((3 <= i && i <= 7 && i % 2 == 0) || (8 <= i && i <= 11 && i % 2 == 1)){
                total_days += 30;
            }
            else if (i == 2){
                total_days += 29;
            }
        }
    }
    total_days += b;
    date = total_days % 7;
    return day[date];
}
int main()
{
	int a=5,b=24;

	//아래는 테스트 출력을 위한 코드입니다.
	cout<<getDayName(a,b);
}

File: Level 2/2016년/2016년.py
def getDayName(a,b):
    answer = ""
    day = [ 'FRI', 'SAT' , 'SUN', 'MON', 'TUE', 'WED', 'THU']
    total_days = - 1
    date = 0
    for i in range(1,a):
        if i != a:
            if i in [1,3,5,7,8,10]:
                total_days += 31
            elif i in [4,6,9,11]:
                total_days += 30
            elif i == 2:
                total_days += 29
    total_days += b
    date = total_days % 7
    return day[date]

#아래 코드는 테스트를 위한 출력 코드입니다.
print(getDayName(5,24))
